lib/charms/grafana_cloud_integrator/v0/cloud_config_requirer.py

```diff
@@ -96,52 +88,49 @@ def _events(self):
 
     @property
     def credentials(self):
```

Comment:
maybe the docstring should mention that the credentials aren't guaranteed to be empty

Reply:
Added docstring